src/test/rustdoc-ui/intra-links-warning.stderr

```diff
@@ -0,0 +1,6 @@
+warning: [src] cannot be resolved, ignoring it...
```

Comment:
What's with this src thing?

Reply:
That's actually a good question... Looking where it's coming from.